docs/testnet/onomy-testnet-docs/setting-up-a-validator-manual.md

```diff
@@ -0,0 +1,255 @@
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/gbt
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/geth
+cd ..
+chmod -R +x binaries
+export PATH=$PATH:$HOME/binaries/
+
+
+or If you have Fedora (Fedora 34) or Redhat (Red Hat Enterprise Linux 8.4 (Ootpa))
+ and you want to make binaries yourself, then follow these steps
+
+sudo yum install -y git
+git clone -b ONET-65 https://github.com/sunnyk56/market.git
+cd market/deploy/onomy-chain
+bash bin.sh
+```
+
+At specific points during the testnet you may be told to 'update your orchestrator' or 'update your onomyd binary'. In order to do that you can simply repeat the above instructions and then restart the affected software.
+
+to check what version of the tools you have run `gbt --version`
+
+### Generate your key
+
+Be sure to back up the phrase you get! You’ll need it in a bit. If you don't back up the phrase here just follow the steps again to generate a new key.
+
+Note 'myvalidatorkeyname' is just the name of your key here, you can pick anything you like, just remember it later.
+
+You'll be prompted to create a password, I suggest you pick something short since you'll be typing it a lot
+
+```
+cd $HOME
+onomyd --home $HOME/onomy/onomy init mymoniker --chain-id onomy
+onomyd --home $HOME/onomy/onomy keys add myvalidatorkeyname --keyring-backend test --output json | jq . >> $HOME/onomy/onomy/validator_key.json
+```
+
+### Copy the genesis file
+
+```
+
+rm $HOME/onomy/onomy/config/genesis.json
+wget http://147.182.128.38:26657/genesis? -O $HOME/raw.json
+jq .result.genesis $HOME/raw.json >> $HOME/onomy/onomy/config/genesis.json
+rm -rf $HOME/raw.json
+```
+
+### Update toml configuration files
+
+Change in $HOME/onomy/onomy/config/config.toml to contain the following:
+
+```
+
+replace seeds = "" to seeds = "1302d0ed290d74d6f061fb8506e0e34f3f67f7ff@147.182.128.38:26656"
+replace "tcp://127.0.0.1:26657" to "tcp://0.0.0.0:26657"
+replace "tcp://127.0.0.1:26656" to "tcp://0.0.0.0:26656"
+replace addr_book_strict = true to addr_book_strict = false
+replace external_address = "" to external_address = "tcp://0.0.0.0:26656"
+```
+
+Change in $HOME/onomy/onomy/config/app.toml to contain the following:
+
+```
+replace enable = false to enable = true
+replace swagger = false to swagger = true
+```
+
+### Start your full node in another terminal and wait for it to sync
+```
+onomyd --home $HOME/onomy/onomy start
+```
+
+### Request some funds be sent to your address
+
+First find your address
+
+```
+
+onomyd --home $HOME/onomy/onomy keys list --keyring-backend test
+
+```
+
+Copy your address from the 'address' field and paste it into the command below in place of $ONOMY_VALIDATOR_ADDRESS
+
+```
+curl -X POST http://147.182.128.38:8000/ -H  "accept: application/json" -H  "Content-Type: application/json" -d "{  \"address\": \"$ONOMY_VALIDATOR_ADDRESS\",  \"coins\": [    \"100000000nom\"  ]}"
```

Comment:
When we have initiated chain and that time we have set genesis coin `"1000000000000nom,1000000000000footoken"`. So have 1000000000000nom in one account and set for the faucet. currently, there is no limit on the faucet in request but we can do that by setting `-max-credit` flag.

but How many times request call this command for an account, currently I have no idea to restrict that one.

Reply:
Let's have a few accounts with the max amount of tokens and limit that is enough to become a validator. In case the validators fetch entire balance we will be able to send more. But don't forget to keep/share all the keys.

Reply:
all above suggestion has been implemented